Source/interfaces/CMakeLists.txt

```diff
@@ -111,14 +111,9 @@ install(
         LIBRARY DESTINATION lib/${NAMESPACE_LIB}/proxystubs COMPONENT libs      # shared lib
 )
 
-install(
-        FILES ${JSON_DATA_HEADERS}
```

Comment:
With the JSON_DATA_HEADERS now not used anymore please remove the file GLOB on line 37 setting JSON_DATA_HEADERS.

Reply:
Agree. Line removed